MTA10/mods/shared_logic/lua/CLuaFunctionDefs.Output.cpp

```diff
@@ -227,7 +227,7 @@ int CLuaFunctionDefs::CreateTrayNotification ( lua_State* luaVM )
     bool useSound = true;
     CScriptArgReader argStream ( luaVM );
     argStream.ReadString ( strText );
-    argStream.ReadString ( strType, "noicon" );
+    argStream.ReadString ( strType, "default" );

```

Comment:
Ok, no problem. I was looking at the enum code earlier, but I wasn't sure if that was required.
@qaisjp